test/e2e/tests/infra/config.go

```diff
@@ -0,0 +1,32 @@
+package infra
```

Comment:
I think these can go into Infra package itself and keep same var checking method currently used in E2E tests